Geometry/VeryForwardGeometryBuilder/plugins/CTPPSGeometryESModule.cc

```diff
@@ -113,6 +115,7 @@ CTPPSGeometryESModule::CTPPSGeometryESModule(const edm::ParameterSet& iConfig)
 void CTPPSGeometryESModule::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.addUntracked<unsigned int>("verbosity", 1);
+  desc.addUntracked<bool>("isRun2", false)->setComment("Switch to legacy (2017-18) definition of diamond geometry");
```

Comment:
Should this parameter be tracked (it affects physics, no?)?

Reply:
Good point. Indeed it can be tracked (however this only affects run 2 configurations, so I believed such a 'legacy' behaviour could be hidden to the new configuration developments).
But maybe another FW expert can comment further?

Reply:
Why not make it tracked?